### PR TITLE
Improvement: limit font scaling for more consistent look

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Closes https://github.com/xbmc/Official-Kodi-Remote-iOS/issues/1012.

Use `minimumScaleFactor` 1.0 if no scaling desired. Use 0.9 as default where different font sizes might disturb, e.g. for library/playlist views. Use 0.8 where different font sizes are not that disturbing.

Screenshots (left = before, right = new)
<a href="https://ibb.co/HgDPNLZ">Bildschirmfoto-2024-06-23-um-17-30-20</a>

## Summary for release notes

Improvement: limit font scaling for more consistent look

### CODE REVIEW

**Comment by wutschel** [comment]
Needs to be rebase after #1087 was merged.

File: XBMC Remote/AppDelegate.h [hunk 1]
```diff
@@ -64,6 +64,11 @@
 
 #define FULLSCREEN_LABEL_HEIGHT 20
 
+/* Font scaling */
+#define FONT_SCALING_MIN 0.8
+#define FONT_SCALING_DEFAULT 0.9
+#define FONT_SCALING_NONE 1.0
```
Comment (kambala-decapitator) [inline XBMC Remote/AppDelegate.h:70]: not a great place for these constants, better move to a dedicated header. Maybe call it Constants.h (or so)

**Review by kambala-decapitator — APPROVED** [review]

File: XBMC Remote/AppDelegate.h (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/AppDelegate.h:70]: The already existing ConvenienceMacros.h is also ok?

Comment (kambala-decapitator) [reply XBMC Remote/AppDelegate.h:70]: yes, I think we created that file exactly for that purpose :)

Comment (wutschel) [reply XBMC Remote/AppDelegate.h]: Done, squashed and rebased.

**Pull request merged**